tests/fe/fe_data_global_test.cc

```diff
@@ -0,0 +1,114 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 1998 - 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+
+#include <deal.II/fe/fe_base.h>
+
+#include <iostream>
+
+#include "../tests.h"
+
+int
+main()
+{
+  std::cout << "2d elements: \n";
```

Comment:
here and below `std::endl`

Reply:
I would add 

```C++
int
main()
{
   init_log();
}
```
and then replace all `std::cout` with `deallog`.